libs/utils/include/utils/vector.h

```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  */
 
-#include <vector>
-
 #ifndef UTILS_VECTOR_H
 #define UTILS_VECTOR_H
 
+#include <vector>
```

Comment:
Why is this still necessary? Doesn't this default the whole purpose?

Reply:
Oh, I see `std::vector` is still used elsewhere in the file.. I'm guessing a later PR will handle switching over to the new `vector`.

Reply:
This file contains utility functions for std::vector too.

Reply:
We can't just blindly switch either because utils::vector works only for trivial types, also I won't be able to make it work with STL allocators, because of how they work. They'll work with our filament allocator though.